app/celery/research_mode_tasks.py

```diff
@@ -205,18 +205,18 @@ def ses_notification_callback(reference):
     }
 
 
-def ses_hard_bounce_callback(reference):
-    return _ses_bounce_callback(reference, 'Permanent')
+def ses_hard_bounce_callback(reference, bounce_subtype=None):
+    return _ses_bounce_callback(reference, 'Permanent', bounce_subtype)
 
 
-def ses_soft_bounce_callback(reference):
-    return _ses_bounce_callback(reference, 'Temporary')
+def ses_soft_bounce_callback(reference, bounce_subtype=None):
+    return _ses_bounce_callback(reference, 'Transient', bounce_subtype)
```

Comment:
`Temporary` was not the right status before, see https://docs.aws.amazon.com/ses/latest/DeveloperGuide/event-publishing-retrieving-sns-contents.html#event-publishing-retrieving-sns-contents-bounce-object